llvm/lib/Target/PowerPC/PPCISelDAGToDAG.cpp

```diff
@@ -7566,6 +7566,100 @@ static void reduceVSXSwap(SDNode *N, SelectionDAG *DAG) {
   DAG->ReplaceAllUsesOfValueWith(SDValue(N, 0), N->getOperand(0));
 }
 
+// Is an ADDI eligible for folding for non-TOC-based local-exec accesses?
+static bool isEligibleToFoldADDIForLocalExecAccesses(SDNode *N,
```

Comment:
I can not find the parameter N be used in the function. do I miss some code ?

Reply:
You're right, I don't think it is needed. Thank you, Digger!